src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1086,7 +1104,9 @@ namespace winrt::TerminalApp::implementation
         const auto focusedTabIndex = _GetFocusedTabIndex();
         const auto control = _GetActiveControl();
         const auto termHeight = control.GetViewHeight();
-        _tabs[focusedTabIndex]->Scroll(termHeight * delta);
+
+        auto tab = winrt::get_self<implementation::Tab>(_tabs.GetAt(focusedTabIndex));
```

Comment:
i believe the vector still has a `[]` operator you can use instead of GetAt -- not 100% sure tho

Reply:
It doesn't seem to work unfortunately 😢